scripts/index.js

```diff
@@ -24,5 +24,22 @@ function squareNumbers(numbers) {
 
 squareNumbers(arrayOne)
 
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 }
+];
 
-function cities
\ No newline at end of file
+function coolCities(citiesArray) {
+    let cities = (citiesArray)
+    let coldCities = []
+    cities.forEach(function (city) {
+    let citytemp = city.temperature
+    if (citytemp < 70) {
+        coldCities.push(city)
+    }
+})
```

Comment:
Not that JavaScript cares, but maybe double check your indentation to improve readability?